src/meta/client/MetaClient.cpp

```diff
@@ -1140,6 +1140,50 @@ folly::Future<StatusOr<bool>> MetaClient::balanceLeader() {
     return future;
 }
 
+folly::Future<StatusOr<std::string>> MetaClient::getTagDefaultValue(GraphSpaceID spaceId,
+                                                                    TagID tagId,
+                                                                    const std::string& field) {
+    cpp2::GetReq req;
+    static std::string defaultKey = "__default__";
```

Comment:
the default value maybe can cached in client.

Reply:
Yep Good Point ~